### ISSUE TITLE
PhantomJS and ".includes" on an array/string

### ISSUE THREAD

**Issue by BrandonCopley** [ariya/phantomjs#14608]
1. Which version of PhantomJS are you using? Tip: run `phantomjs --version`.
2.1.1

2. What steps will reproduce the problem?
  1. This line of javascript - if ("a string of any length that has /#!/ in it".includes("/#!/")){}

3. Which operating system are you using?
Mac OSX

4. Did you use binary PhantomJS or did you compile it from source?
I used the binary form, this also broke on prerender.io

5. Please provide any additional information below.
Basically the ".includes" on a string throws errors when phantomjs is building the website. It looks like perhaps phantomjs doesn't understand that line?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

**Comment by Vitallium** [comment]
This will be fixed in 2.5. See #14506
